tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
   "compilerOptions": {
     "target": "es2017",
-    "rootDir": "src",
```

Comment:
you can keep this as we are not going to generate code anymore, all code will be under `src`

Reply:
fixed in c5dd8fc